src/packet/public_key_encrypted_session_key.js

```diff
@@ -180,9 +180,10 @@ class PublicKeyEncryptedSessionKeyPacket {
     // No symmetric encryption algorithm identifier is passed to the public-key algorithm for a
     // v6 PKESK packet, as it is included in the v2 SEIPD packet.
     const sessionKeyAlgorithm = this.version === 3 ? this.sessionKeyAlgorithm : null;
+    const fingerprint = this.version === 6 ? key.getFingerprintBytes() : key.getFingerprintBytes().subarray(0, 20);
```

Comment:
I would make v5 the special case, rather than v6; that way, v7 is more likely to work out of the box ^.^
```suggestion
    const fingerprint = this.version !== 5 ? key.getFingerprintBytes() : key.getFingerprintBytes().subarray(0, 20);
```
(v4 keys have 20 byte fingerprints so both branches work there)